cmd/kubelet/app/server.go

```diff
@@ -194,6 +198,7 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&s.MaxContainerCount, "maximum-dead-containers", s.MaxContainerCount, "Maximum number of old instances of a containers to retain globally.  Each container takes up some disk space.  Default: 100.")
 	fs.StringVar(&s.AuthPath, "auth-path", s.AuthPath, "Path to .kubernetes_auth file, specifying how to authenticate to API server.")
 	fs.MarkDeprecated("auth-path", "will be removed in a future version")
+	fs.StringVar(&s.KubeConfig, "kubeconfig", s.KubeConfig, "Path to a kubeconfig file, specifying how to authenticate to API server.")
```

Comment:
Do you ever want to respect the server listed in the kubeconfig file?  If not, add that information to the description here.